data/emitter

```diff
@@ -1,3 +1,9 @@
-{"t":1482188362433,"m":"Workspace created in /var/folders/d6/rwx8bw4s4773s5v75z185j4r001ttj/T/ArtifactDir701279641","s":"sd-setup-launcher"}
-{"t":1482188362433,"m":"Source Dir: /var/folders/d6/rwx8bw4s4773s5v75z185j4r001ttj/T/ArtifactDir701279641/src/github.com/screwdriver-cd/launcher","s":"sd-setup-launcher"}
-{"t":1482188362433,"m":"Artifacts Dir: /var/folders/d6/rwx8bw4s4773s5v75z185j4r001ttj/T/ArtifactDir701279641/artifacts","s":"sd-setup-launcher"}
+{"t":1551828841797,"m":"Screwdriver Launcher information","s":"sd-setup-launcher"}
```

Comment:
Can commit this but the content of this file doesn't matter. 🤷‍♀️ 
Each time the test runs, it will generate a new file.